src/OpenTelemetry.Resources.AssemblyMetadata/OpenTelemetry.Resources.AssemblyMetadata.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>$(NetMinimumSupportedVersion);$(NetStandardMinimumSupportedVersion)</TargetFrameworks>
+    <Description>OpenTelemetry Resource Detectors for assembly metadata.</Description>
+    <MinVerTagPrefix>Resources.AssemblyMetadata-</MinVerTagPrefix>
+  </PropertyGroup>
+
+  <!-- Do not run Package Baseline Validation as this package has never released a stable version.
+  Remove this property once we have released a stable version and add PackageValidationBaselineVersion property. -->
+  <PropertyGroup>
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(RepoRoot)\src\Shared\Guard.cs" Link="Includes\Guard.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="_.targets" Pack="true" PackagePath="buildTransitive\$(TargetFramework)\$(AssemblyName).targets" />
+  </ItemGroup>
```

Comment:
I feel it's adding unnecessary complexity to define a new item type to re-use SDK-built-in functionality (i.e. the `AssemblyMetdata` item).